### PR TITLE
trivial: various superficial fixes

### PR DESCRIPTION
**Pull request by tomgg**
Here are some harmless changes I made while getting familiar with the codebase.

### CODE REVIEW

File: core/commands/commands.go [hunk 1]
```diff
@@ -1,10 +1,10 @@
-/*
-Package commands implements the IPFS command interface
-
-Using github.com/ipfs/go-ipfs/commands to define the command line and
-HTTP APIs.  This is the interface available to folks consuming IPFS
-from outside of the Go language.
-*/
+//
```
Comment (whyrusleeping) [inline core/commands/commands.go]: leave off the empty comment lines before and after

File: commands/command.go [hunk 2]
```diff
@@ -217,12 +218,15 @@ func (c *Command) CheckArguments(req Request) error {
 	// iterate over the arg definitions
 	valueIndex := 0 // the index of the current value (in `args`)
 	for _, argDef := range c.Arguments {
-		// skip optional argument definitions if there aren't sufficient remaining values
-		if len(args)-valueIndex <= numRequired && !argDef.Required || argDef.Type == ArgFile {
+		// skip optional argument definitions if there aren't
+		// sufficient remaining values
+		if len(args)-valueIndex <= numRequired && !argDef.Required ||
+		argDef.Type == ArgFile {
```
Comment (whyrusleeping) [inline commands/command.go]: did you run gofmt? i'm not sure this is the right indentation here.

Comment (tomgg) [inline commands/command.go]: Jeromy Johnson <reply@reply.github.com> / 2016-02-02 21:30-0800
> > @@ -217,12 +218,15 @@ func (c *Command) CheckArguments(req Request) error {
> >  	// iterate over the arg definitions
> >  	valueIndex := 0 // the index of the current value (in `args`)
> >  	for _, argDef := range c.Arguments {
> > -		// skip optional argument definitions if there aren't sufficient remaining values
> > -		if len(args)-valueIndex <= numRequired && !argDef.Required || argDef.Type == ArgFile {
> > +		// skip optional argument definitions if there aren't
> > +		// sufficient remaining values
> > +		if len(args)-valueIndex <= numRequired && !argDef.Required ||
> > +		argDef.Type == ArgFile {
> 
> did you run gofmt? i'm not sure this is the right indentation here.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ipfs/go-ipfs/pull/2288/files#r51680222

gofmt++ (hadn't heard of it until now---what a noob, right?)

**Comment by whyrusleeping** [comment]
alright cool, this looks good to me, thanks!

**Pull request merged**